src/window.rs

```diff
@@ -1379,3 +1376,17 @@ impl Default for UserAttentionType {
         UserAttentionType::Informational
     }
 }
+
+/// Describes the window order in z-hierarchy.
```

Comment:
Something more descriptive imo - trying to emphasize the grouping aspect of the window levels

```suggestion
/// A window level groups windows with respect to their z-position.
///
/// The relative ordering between windows in different window levels is fixed.
/// The z-order of a window within the same window level may change dynamically on user interaction.
```

Reply:
thanks, I really didn't know what to write there.